### ISSUE TITLE
ERR_CONNECTION_REFUSED

### ISSUE THREAD

**Issue by JAG007** [magenx/Magento-2-server-installation#13]
I am getting this error in the browser:

This site can’t be reached
www.domain.com refused to connect.

I have followed the instructions to the letter:

I managed to get the script to work earlier before you made the changes to fix the cerbot issue but I can not figure out what is wrong now.

Thanks
Any help would be most appreciated

**Comment by JAG007** [comment]
i have followed the setup:

I have NOT used the minimal magento.
and NOT changed the ssh port.
I have used the given database and database user in the database section.
In the install section ->  Enter your shop url: I have used both https and http with no effect on the result.
I have entered one domain without www. in the config section (the same as my shop domain in magento section).
I have answered yes to both request to rewrite the files in the config section.
I have updated the /etc/varnish/default.vcl to remove /pub .
I have even tried changing all permissions to 777 to see if it was a permissions issue but no effect on the result.

I am still getting 

**ERR_CONNECTION_REFUSED**

**This site can’t be reached
www.domain.com refused to connect.**
